consensus/hotstuff/signature.go

```diff
@@ -8,6 +8,10 @@ import (
 // RandomBeaconReconstructor collects verified signature shares, and reconstructs the
 // group signature with enough shares.
 type RandomBeaconReconstructor interface {
+	// Verify returns true if and only if the signature is valid.
+	// It expects that correct type of signature is passed. Only SigTypeRandomBeacon is supported
+	Verify(signerID flow.Identifier, sig crypto.Signature) (bool, error)
```

Comment:
```suggestion
	Verify(signerID flow.Identifier, sig crypto.Signature) error
```

We could use different sentinal errors for different reasons of why the sig is invalid.

Reply:
I would leave it as it is, just to align with other `Verify` functions.